src/components/AccountCard.vue

```diff
@@ -137,7 +137,7 @@ export default defineComponent({
 
         let profile = computed(() => profileStore.profiles.get(props.account));
 
-
+        const showProfileData = computed(() => profileData.value.displayName !== '' || profileData.value.status !== '' || profileData.value.bio !== '');
```

Comment:
Maybe we should hide profile info if all fields are empty... this line will hide everything if only one is empty...

Reply:
I think the logic is to show the profile data if at least of the attributes is different from empty string

Reply:
Yep, sorry... you are right